cmd/server/reader.go

```diff
@@ -8,40 +8,47 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 
 	"github.com/moov-io/base/log"
 	"github.com/moov-io/fed"
 	"github.com/moov-io/fed/pkg/download"
 )
 
 func fedACHDataFile(logger log.Logger) (io.Reader, error) {
-	if file, err := attemptFileDownload(logger, "fedach"); file != nil {
-		return file, nil
-	} else if err != nil {
+	file, err := attemptFileDownload(logger, "fedach")
+	if err != nil && !strings.Contains(err.Error(), download.MissingRoutingNumberErr) && !strings.Contains(err.Error(), download.MissingDownloadCodeErr) {
 		return nil, fmt.Errorf("problem downloading fedach: %v", err)
 	}
 
+	if file != nil {
+		return file, nil
+	}
+
 	path := readDataFilepath("FEDACH_DATA_PATH", "./data/FedACHdir.txt")
 	logger.Logf("search: loading %s for ACH data", path)
 
-	file, err := os.Open(path)
+	file, err = os.Open(path)
 	if err != nil {
 		return nil, fmt.Errorf("problem opening %s: %v", path, err)
 	}
 	return file, nil
 }
 
 func fedWireDataFile(logger log.Logger) (io.Reader, error) {
-	if file, err := attemptFileDownload(logger, "fedwire"); file != nil {
+	file, err := attemptFileDownload(logger, "fedach")
+	if err != nil && !strings.Contains(err.Error(), download.MissingRoutingNumberErr) && !strings.Contains(err.Error(), download.MissingDownloadCodeErr) {
+		return nil, fmt.Errorf("problem downloading fedach: %v", err)
+	}
```

Comment:
We could try to return a common `ErrMissingData = errors.New("missing data")` and check it a bit easier `errors.Is(err, ErrMissingData)`.

Reply:
I was having trouble getting `errors.Is` to work so I tried this approach instead. I figured out that the error wasn't wrapped and that was causing `errors.Is` to fail to match. Now that I've fixed that, do we still want a single error string for missing download code, routing number, and download url template?

Reply:
A common configuration error can be wrapped with specific details if we want. That way the caller can just check for the common error, but we log more details for the user.